code/modules/uplink/uplink_items.dm

```diff
@@ -120,6 +120,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	var/list/include_modes = list() // Game modes to allow this item in.
 	var/list/exclude_modes = list() // Game modes to disallow this item from.
 	var/list/restricted_roles = list() //If this uplink item is only available to certain roles. Roles are dependent on the frequency chip or stored ID.
+	var/list/restricted_traits = list() //If this uplink item is only available to certain traits.
```

Comment:
```suggestion
	var/list/required_traits = list() //If this uplink item is only available to certain traits.
```
restricted sounds like you can't have those traits

Reply:
But then they're all equally wrong!